Makefile

```diff
@@ -263,6 +267,35 @@ image.%:
 	NRI_IMAGE_TAR=`realpath "$(IMAGE_PATH)/$${tag}-image-$${NRI_IMAGE_ID}.tar"`; \
 	$(DOCKER) image save "$${NRI_IMAGE_REPOTAG}" > "$${NRI_IMAGE_TAR}";
 
+.PHONY: licenses
+licenses: $(foreach p,$(PLUGINS),licenses.$(p)) $(foreach p,$(BINARIES),licenses.$(p))
+
+licenses.nri-resource-policy-% \
+licenses.nri-% \
+licenses.%:
+	$(Q)mkdir -p $(LICENSE_PATH); \
+	case $@ in \
+	    *.nri-resource-policy-*) \
+		dir=$(patsubst licenses.nri-resource-policy-%,./cmd/plugins/%,$@); \
+	        ;; \
+	    *.nri-*) \
+		dir=$(patsubst licenses.nri-%,./cmd/plugins/%,$@); \
+	        ;; \
+	    *.*) \
+		dir=$(patsubst licenses.%,./cmd/%,$@); \
+	        ;; \
+	esac; \
+	bin="$(patsubst licenses.%,%,$@)"; \
+	out="$(LICENSE_PATH)/$$bin"; \
+	rm -fr "$$out" && \
+	go-licenses save "$$dir" \
+	    --ignore github.com/containers/nri-plugins \
```

Comment:
I'm scratching my head wondering why this directory was excluded.

Reply:
It's not about the directory, it's about skipping the nri-plugins package itself. But should we include this as well(?)

Reply:
> I'm scratching my head wondering why this directory was excluded.

Originally it was (I think) because pkg/topology lacked a license but was a go module of it's own, which go-licenses could not handle. I fixed that locally, but forgot to (fully) commit it...